source/system-administrators/deployer/processors-guide.rst

```diff
@@ -63,33 +68,35 @@ Processor that clones/pulls a remote Git repository into a local path in the fil
-|            |           |                               |key pair authentication                                     |
-+------------+-----------+-------------------------------+------------------------------------------------------------+
++------------+-----------+-------------------------------+-------------------------------------------------------------+
+|Name        |Required   |Default Value                  |Description                                                  |
++============+===========+===============================+=============================================================+
+||URL|       ||checkmark||                               |The URL of the remote Git repo to pull                       |
++------------+-----------+-------------------------------+-------------------------------------------------------------+
+||Name|      |           |``origin``                     |The name to use for the remote repo when pulling from it     |
++------------+-----------+-------------------------------+-------------------------------------------------------------+
+||Branch|    |           |The default branch in the repo |The branch of the remote Git repo to pull                    |
++------------+-----------+-------------------------------+-------------------------------------------------------------+
+||username|  |           |                               |The username for authentication with the remote Git repo.    |
+|            |           |                               |Not needed when SSH with RSA key pair authentication is used |
++------------+-----------+-------------------------------+-------------------------------------------------------------+
+||password|  |           |                               |The password for authentication with the remote Git repo.    |
+|            |           |                               |Not needed when SSH with RSA key pair authentication is used |
++------------+-----------+-------------------------------+-------------------------------------------------------------+
+||path|      |           |                               |The SSH private key path, used only with SSH with RSA key    |
+|            |           |                               |pair authentication                                          |
++------------+-----------+-------------------------------+-------------------------------------------------------------+
+||passphrase||           |                               |The SSH private key passphrase, used only with SSH withRSA   |
+|            |           |                               |key pair authentication                                      |
++------------+-----------+-------------------------------+-------------------------------------------------------------+
+||failDep|   |           |``true``                       |Enables failing a deployment when there's a processor failure|
```

Comment:
Do we need to repeat it? The first table includes all common properties

Reply:
The default value for the property listed in the common table is `false` and I wasn't sure how to show users that the default for that property for certain processors is `true`...

Reply:
got it, makes sense :+1: